.github/renovate.json5

```diff
@@ -32,6 +32,7 @@
         '^example/.+\\.yaml$',
         '^hack/.+\\.yaml$',
         '^\\.test-defs/.+\\.yaml$',
+        '^charts/.+\\.yaml$',
```

Comment:
This includes YAML files in the `charts` folder for Renovate to update `image` tags such as this one:
[`2f63ea8` (#10224)](https://github.com/gardener/gardener/pull/10224/commits/2f63ea8c9a44f80dbd2c65d2d6136f92440144c3)

Regex test:
https://regex101.com/r/dvfQdb/1

Reply:
/cc @oliver-goetz

Reply:
You could also remove [this line](https://github.com/gardener/gardener/blob/9278efdb615dbfad113afe67f4afe53f6dd8c9da/.github/renovate.json5#L380). It was meant to prevent false positives in chart updates. Apparently, it does not prevent anything anyway as there is a typo in the folder name 😅

Reply:
Fair enough and well spotted :D 
Done via [`3e48f0a` (#10909)](https://github.com/gardener/gardener/pull/10909/commits/3e48f0a8273da503e8ee194eb7a51313782fb439)